.circleci/config.yml

```diff
@@ -108,7 +109,15 @@ jobs:
           working_directory: /home/circleci
           command: |
             echo "Master theme branch is ${CIRCLE_BRANCH}"
-            MASTER_THEME_BRANCH=dev-${CIRCLE_BRANCH} MERGE_SOURCE=git@github.com:greenpeace/planet4-base-fork.git MERGE_REF=develop make
+            if [ -n "$CIRCLE_PR_NUMBER" ]; then
+              BRANCH="$(./get-fork-pr-branch.sh)"
+            else
+              BRANCH="${CIRCLE_BRANCH}"
+            fi
+            MASTER_THEME_BRANCH=dev-${BRANCH}#${CIRCLE_SHA1} \
+            MERGE_SOURCE=git@github.com:greenpeace/planet4-base-fork.git \
+            MERGE_REF=develop \
+            make ci
```

Comment:
Is it intentional that this goes from `make` to `make ci`?

Reply:
Yes, it's needed to run the tests without secrets. `make` has the `push` step at the end (https://github.com/greenpeace/planet4-builder/blob/846861ee8d3dfb2a984997082d5677e73a10b395/src/Makefile#L162-L162), whereas `make ci` doesn't (https://github.com/greenpeace/planet4-builder/blob/846861ee8d3dfb2a984997082d5677e73a10b395/src/Makefile#L166-L166)

Reply:
In the blocks plugin we actually already use `make ci` in the same place https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/c9b268862b8bb2a62b68893751e5721353fc785c/.circleci/config.yml#L121-L121